src/validation.cpp

```diff
@@ -247,6 +247,12 @@ bool CheckSequenceLocksAtTip(CBlockIndex* tip,
                 }
             }
             lp->maxInputBlock = tip->GetAncestor(maxInputHeight);
+            // tip->GetAncestor(maxInputHeight) should never return a nullptr
+            // because maxInputHeight is always less than the tip height.
+            // It would, however, be a bad bug to continue execution, since a
+            // LockPoints object with the maxInputBlock member set to nullptr
+            // signifies no relative lock time.
+            assert(lp->maxInputBlock);
```

Comment:
```diff
-            lp->maxInputBlock = tip->GetAncestor(maxInputHeight);
             // tip->GetAncestor(maxInputHeight) should never return a nullptr
             // because maxInputHeight is always less than the tip height.
             // It would, however, be a bad bug to continue execution, since a
             // LockPoints object with the maxInputBlock member set to nullptr
             // signifies no relative lock time.
-            assert(lp->maxInputBlock);
+            lp->maxInputBlock = Assert(tip->GetAncestor(maxInputHeight));
```